internal/ingest/ingest.go

```diff
@@ -137,9 +136,11 @@ type Ingester struct {
 
 	// Multihash minimum length
 	minKeyLen int
-
-	// entryWP is a worker pool for asynchronous calls to indexer.Put.
-	entryWP *workerpool.WorkerPool
+	// syncWriteEntries is accessed through Ingester.SetSyncWriteEntries() and
+	// Ingester.syncWriteEntries(). It tells the Ingester to handle write entry
+	// chunks synchronously, waiting for each to complete before fetching the
+	// next.
+	_syncWriteEntries uint32
```

Comment:
`_` prefix a new naming convention? I don't recall it's use elsewhere.

Reply:
The reason for the `_` prefix is because this member variable should not be accessed directly by things in the same package, but through the functions, `syncWriteEntries()` and `SetSyncWriteEntries()`

Reply:
I recommend adding that as a comment, and update the godoc to use the same name in the beginning.